pyart/util/sigmath.py

```diff
@@ -32,3 +32,38 @@ def texture(myradar, var):
         tex[timestep, 0:4] = np.ones(4) * ray[0]
         tex[timestep, -5:] = np.ones(5) * ray[-1]
     return tex
+
+
+def texture_along_ray(myradar, var, wind_size=7, min_valid=3):
+    """
+    Compute field texture along ray using a user specified
+    window size. If the number of valid range bins within the
+    window is lower than min_valid the bin is set to invalid.
```

Comment:
The descriptions of this function does not match what is being done in the code.  The most obvious difference is that `min_valid` is not used at all.  Either to documentation or the code should be updated.